lesson_07/objects/objects_app/src/main/java/com/codedifferently/lesson7/richhawkins/House.java

```diff
@@ -0,0 +1,61 @@
+package com.codedifferently.lesson7.richhawkins;
+
+import java.time.Year;
+
+public class House {
+  //     The class must have at least 5 member variables of at least 3 different types.
+  String streetName;
+  int streetNum;
+  int yearBuilt;
```

Comment:
You're not using the `yearBuilt`, `hasGarage`, and `numberOfBedrooms` members. You should create and test property accessors for them or remove.

Reply:
All member variables now used with property accessors (learned what that meant) and tests are included for them.